### PR TITLE
#308: add permissions to GenericViews

### PR DESCRIPTION
**Pull request by vladarefiev**
PR for https://github.com/timelesslounge/timelessis/issues/308:
- added permissions for GenericViews
- added `auth.login_required` to `ItemCreateView` as test
- added tests for that

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#477](https://github.com/timelesslounge/timelessis/issues/477) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#477](https://github.com/timelesslounge/timelessis/issues/477) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/80f61baa-41cb-42a5-ae1a-007118bee880); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: tests/view/items/test_items.py [hunk 1]
```diff
@@ -55,6 +54,15 @@ def test_create(client):
     assert database_item.employee_id == employee.id
 
 
+@pytest.mark.parametrize("path", (
+    "/items/create",
```
Comment (vryazanov) [inline tests/view/items/test_items.py]: @vladarefiev mb we will use `url_for` instead of hardcoded urls?

Comment (vladarefiev) [reply tests/view/items/test_items.py]: @vryazanov can't find solution, i have faced with 
```
RuntimeError: Attempted to generate a URL without the application context being pushed. This has to be executed when application context is available.
```
Do you know smth about this one?

**Comment by vladarefiev** [comment]
@vryazanov fixed, have a look pls

File: tests/view/items/test_items.py (hunk 1 above)
Comment (vladarefiev) [reply tests/view/items/test_items.py]: @vryazanov fixed

**Comment by vryazanov** [comment]
@rultor merge

**Comment by emilianodellacasa** [comment]
@vladarefiev you left single quotes in the code please check

**Comment by vladarefiev** [comment]
@emilianodellacasa done, pls take a look

**Comment by emilianodellacasa** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job was finished in 14 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vryazanov[/z](https://www.0crat.com/u/vryazanov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#477](https://github.com/timelesslounge/timelessis/issues/477) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@vryazanov According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by vryazanov** [comment]
@ypshenychka confirm

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +20 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Pull request merged**